go.mod

```diff
@@ -79,4 +79,5 @@ replace (
 	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190817221809-bf4de9df677c
 	k8s.io/client-go => k8s.io/client-go v0.0.0-20190817222206-ee6c071a42cf
 	k8s.io/klog => github.com/mikkeloscar/knolog v0.0.0-20190326191552-80742771eb6b
+	github.com/envoyproxy/go-control-plane => github.com/envoyproxy/go-control-plane v0.8.7-0.20190821215049-f062b07a671a
```

Comment:
can we remove the `replace` directives for the k8s deps?

Reply:
(Not really required) If not possible, we can leave it as-is. I can address it in my PR

Reply:
I added the `go-control-plane` one since Contour has a later version in it's go.mod than Istio. When compiling there was a diff issue and couldn't get resolved. I'm happy to remove, this was just to make everything build properly.

Reply:
Oh got it! Makes sense :)